pkg/clients/builder.go

```diff
@@ -75,6 +77,9 @@ func (b *Builder) BuildOrDie(ctx context.Context) *Clients {
 	controlPlaneDynamicClient := dynamic.NewForConfigOrDie(controlPlaneRestConfig)
 	controlPlaneDynamicInformers := dynamicinformer.NewFilteredDynamicSharedInformerFactory(controlPlaneDynamicClient, b.resync, b.controllerConfig.OperatorNamespace, nil)
 
+	controlPlaneHypeClient := hypextclient.NewForConfigOrDie(controlPlaneRestConfig)
+	controlPlaneHypeInformers := hypextinformers.NewFilteredSharedInformerFactory(controlPlaneHypeClient, b.resync, b.controllerConfig.OperatorNamespace, nil)
+
```

Comment:
This should be initialized only on HyperShift. Otherwise the HyperShift CRDs are not in the API server and esp. the informer will log weird errors periodically.